Client/Frontend/Browser/TabTrayButtonExtensions.swift

```diff
@@ -7,6 +7,11 @@ import UIKit
 import Shared
 
 class PrivateModeButton: ToggleButton, PrivateModeUI {
+    
+    // Ecosia: Add tints
+    private var offTint = UIColor.black
+    private var onTint = UIColor.black
```

Comment:
Are those two variables being used?

Reply:
Well spotted! Eventually not used with the latest changes. Removed!